template-for-repository/proofs/Makefile.common

```diff
@@ -402,6 +402,7 @@ COMMA :=,
 # Set C compiler defines
 
 CBMCFLAGS += --object-bits $(CBMC_OBJECT_BITS)
```

Comment:
Does `--object-bits` need to be passed to both `goto-cc` and `cbmc` or is is sufficient to pass it only to `goto-cc`? It may be safer to use this parameter only once.

Reply:
Good point, @remi-delmas-3000! I'd expect that we only need to pass it during compilation with `goto-cc`. @tautschnig any insights on this?

Reply:
@remi-delmas-3000 based on an offline discussion, it appears we must pass the same value for both invocations.